packages/apps/backend/src/api-server/ApiServer.ts

```diff
@@ -45,15 +46,27 @@ export class ApiServer extends EventEmitter<ApiServerEvents> {
 		super()
 		this.log = log.category('ApiServer')
 
-		this.app.use(serveStatic('src'))
+		const serveStaticMiddleware = serveStatic(fileURLToPath(new URL('../../../client/dist', import.meta.url)))
+		this.app.use(serveStaticMiddleware)
 
 		this.app.use(
 			cors({
 				origin: '*', // TODO: cors
 			})
 		)
 
-		this.app.use(errorHandler())
+		const errorHandlerMiddleware = errorHandler()
+		this.app.use((ctx: FeathersKoaContext, next) => {
+			if (ctx.path.startsWith('/api/') || ctx.path.startsWith('/api')) {
```

Comment:
Should this be `ctx.path.startsWith('/api')` or should it be `ctx.path === '/api`?

Reply:
Good spot, I meant this to be `ctx.path.startsWith('/api/') || ctx.path === '/api'`, but was being clumsy.

I did start with `ctx.path.startsWith('/api')`, but that means it also caught `/api-not-really` which isnt technically correct